packages/node/package.json

```diff
@@ -42,6 +42,7 @@
         "typescript": "^5.0.4"
     },
     "dependencies": {
-        "form-data": "^4.0.0"
+        "form-data": "^4.0.0",
+        "native-reg": "^1.1.1"
```

Comment:
wouldn't peerDependency force our user to install form-data and native-reg? It would be better to stay with dependencies if I'm correct. I would understand to use peerpackages for express or for any other library that has to exists in this case, but here we're open for everything